src/lib.rs

```diff
@@ -319,6 +321,16 @@ where
     limiter: L,
 }
 
+impl<K, V, L: Limiter<K, V>, S> Debug for LruMap<K, V, L, S> {
+    fn fmt(&self, f: &mut Formatter<'_>) -> core::fmt::Result {
+        f.debug_struct("LruMap")
+            .field("len", &self.map.len())
+            .field("key_type", &type_name::<K>())
+            .field("value_type", &type_name::<V>())
+            .finish()
```

Comment:
I'd probably use this instead since debug output doesn't represent contents of the struct exactly:
```suggestion
            .finish_non_exhaustive()
```